apis/core/test/csvw-builder/index.test.ts

```diff
@@ -198,12 +198,12 @@ describe('lib/csvw-builder', () => {
     csvSource.columns = [
       CsvColumn.fromPointer(csvSource.pointer.namedNode('jahr-column'), { name: 'JAHR' }),
     ]
-    const columnMapping = ColumnMapping.fromPointer(clownface({ dataset: $rdf.dataset() }).namedNode('year-mapping'), {
+    const columnMapping = ColumnMapping.literalFromPointer(clownface({ dataset: $rdf.dataset() }).namedNode('year-mapping'), {
       sourceColumn: $rdf.namedNode('jahr-column') as any,
       targetProperty: schema.yearBuilt,
       language: 'de-DE',
     })
-    resources.push(columnMapping.pointer)
+    resources.push(columnMapping.pointer as any)
```

Comment:
With the change in `TestResourceStore` all those `as any` would be unnecessary?

Reply:
Yeah, it was yelling at me and I couldn't figure out how to fix it